projects/1.20/assets/furnish/furnish/lang/zh_cn.json

```
@@ -0,0 +1,396 @@
+  "block.furnish.lime_curtain": "黄绿色窗帘",
+  "block.furnish.magenta_curtain": "品红色窗帘",
+  "block.furnish.orange_curtain": "橙色窗帘",
+  "block.furnish.pink_curtain": "粉红色窗帘",
+  "block.furnish.purple_curtain": "紫色窗帘",
+  "block.furnish.red_curtain": "红色窗帘",
+  "block.furnish.yellow_curtain": "黄色窗帘",
+  "block.furnish.white_showcase": "白色展柜",
+  "block.furnish.black_showcase": "黑色展柜",
+  "block.furnish.blue_showcase": "蓝色展柜",
+  "block.furnish.brown_showcase": "棕色展柜",
+  "block.furnish.cyan_showcase": "青色展柜",
+  "block.furnish.gray_showcase": "灰色展柜",
+  "block.furnish.green_showcase": "绿色展柜",
+  "block.furnish.light_blue_showcase": "淡蓝色展柜",
+  "block.furnish.light_gray_showcase": "淡灰色展柜",
+  "block.furnish.lime_showcase": "黄绿色展柜",
+  "block.furnish.magenta_showcase": "品红色展柜",
+  "block.furnish.orange_showcase": "橙色展柜",
+  "block.furnish.pink_showcase": "粉红色展柜",
+  "block.furnish.purple_showcase": "紫色展柜",
+  "block.furnish.red_showcase": "红色展柜",
+  "block.furnish.yellow_showcase": "黄色展柜",
+  "block.furnish.crimson_drawers": "绯红木抽屉",
+  "block.furnish.crimson_cabinet": "绯红木柜",
+  "block.furnish.crimson_wardrobe": "绯红木立柜",
+  "block.furnish.crimson_chair": "绯红木椅",
+  "block.furnish.crimson_stool": "绯红木凳",
+  "block.furnish.crimson_table": "绯红木桌",
+  "block.furnish.crimson_square_table": "绯红木方桌",
+  "block.furnish.crimson_pedestal_table": "绯红木柱脚桌",
+  "block.furnish.crimson_shutter": "绯红木百叶窗",
+  "block.furnish.crimson_crate": "绯红木板条箱",
+  "block.furnish.crimson_bench": "绯红木长椅",
+  "block.furnish.crimson_ladder": "绯红木梯子",
+  "block.furnish.crimson_log_bench": "绯红木菌柄椅",
+  "block.furnish.crimson_shelf": "绯红木架",
+  "block.furnish.warped_bench": "诡异木长椅",
+  "block.furnish.warped_log_bench": "诡异木菌柄椅",
+  "block.furnish.warped_crate": "诡异木板条箱",
+  "block.furnish.warped_ladder": "诡异木梯子",
+  "block.furnish.warped_cabinet": "诡异木柜",
+  "block.furnish.warped_chair": "诡异木椅",
+  "block.furnish.warped_drawers": "诡异木抽屉",
+  "block.furnish.warped_pedestal_table": "诡异木柱脚桌",
+  "block.furnish.warped_shelf": "诡异木架",
+  "block.furnish.warped_shutter": "诡异木百叶窗",
+  "block.furnish.warped_square_table": "诡异木方桌",
+  "block.furnish.warped_stool": "诡异木凳",
+  "block.furnish.warped_table": "诡异木桌",
+  "block.furnish.warped_wardrobe": "诡异木立柜",
+  "block.furnish.disk_rack": "唱片架",
+  "block.furnish.heavy_metal": "重金属",
+  "block.furnish.small_locker": "小储物柜",
+  "block.furnish.locker": "储物柜",
+  "block.furnish.white_animal_basket": "白色动物篮",
+  "block.furnish.acacia_bookshelf_chest": "金合欢书柜",
+  "block.furnish.acacia_coffin": "金合欢棺材",
+  "block.furnish.asphalt": "沥青",
+  "block.furnish.bamboo_bedside_table": "竹床头柜",
+  "block.furnish.bamboo_bench": "竹长椅",
+  "block.furnish.bamboo_bookshelf_chest": "竹书柜",
+  "block.furnish.bamboo_cabinet": "竹柜",
+  "block.furnish.bamboo_chair": "竹椅",
+  "block.furnish.bamboo_crate": "竹板条箱",
+  "block.furnish.bamboo_kitchen_cabinet": "竹厨柜",
+  "block.furnish.bamboo_ladder": "竹梯子",
+  "block.furnish.bamboo_log_bench": "竹原木长椅",

```

```suggestion
  "block.furnish.bamboo_log_bench": "竹原木椅",
```
木制品请统一称呼
建议全文检查所有木制品的名称

---

本mod中同时有chair（椅），bench（长椅）和stool（凳），已按原内容进行区分

---

已进行检查